server/xpub-model/entities/file/index.js

```diff
@@ -25,23 +26,33 @@ class File extends BaseModel {
     }
   }
 
-  async save() {
+  async save(userId) {
     await this.$query().upsertGraphAndFetch(this)
```

Comment:
is this the line that actually saves the `File` object?

Reply:
Yep, this is where `objection` sends the db data